app/cu_editor.py

```diff
@@ -32,6 +32,8 @@ def initCommandSet(editText, textBuffer):
 
     CTRL_H: textBuffer.backspace,
 
+    CTRL_L: textBuffer.selectCurrentLine,
```

Comment:
Yes the intent is for it to operate everywhere, since there doesn't yet seem to be an easy way to erase the text written in the prompt UIs at the bottom of the screen.